docs/reference/master-commandline-reference.md

```diff
@@ -149,6 +149,20 @@ nfd-master -verify-node-name -ca-file=/opt/nfd/ca.crt \
     -cert-file=/opt/nfd/master.crt -key-file=/opt/nfd/master.key
 ```
 
+### -enable-nodefeature-api
+
+The `-enable-nodefeature-api` flag enables the NodeFeature CRD API for
+receiving feature requests. This will also automatically disable the gRPC
+interface.
+
+Default: false
+
+Example:
+
+```bash
+nfd-master -enable-nodefeature-api
+```
+
```

Comment:
👍🏽

Reply:
The flags are not organized in alphabetical order but merely in grouped, in an attempt to form some logical groups (e.g. TLS-related flags together etc...). Maybe a alphabetical order would be best, I don't know but that can be addressed separately 🤷‍♀️

I didn't move `-enable-nodefeature-api` over to `-enable-taints` but the other way around (in a separate patch) because I feel that `-enable-taints`, `-enable-nodefeature-api` and `-no-publish` kinda belong together. WDYT?

Reply:
Agreed, let's keep what you have in the last.